<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicBackgroundSkinDialog</class>
 <widget class="QWidget" name="MusicBackgroundSkinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="showPerArea">
   <property name="geometry">
    <rect>
     <x>482</x>
     <y>125</y>
     <width>24</width>
     <height>226</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230,115,0);</string>
   </property>
   <property name="text">
    <string>100%







 50%






 0%</string>
   </property>
  </widget>
  <widget class="QSlider" name="bgTransparentSlider">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>20</width>
     <height>225</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>530</width>
     <height>370</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>皮肤与窗口调整</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mySkin">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>我的皮肤</string>
   </property>
  </widget>
  <widget class="QPushButton" name="netSkin">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>70</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>推荐皮肤</string>
   </property>
  </widget>
  <widget class="QSlider" name="bgTransparentSliderR">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>20</width>
     <height>225</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>120</y>
     <width>395</width>
     <height>235</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="MusicBackgroundListWidget" name="themeListWidget"/>
   <widget class="MusicBackgroundRemoteWidget" name="remoteWidget"/>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>105</y>
     <width>510</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="customSkin">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>自定义皮肤</string>
   </property>
  </widget>
  <widget class="QPushButton" name="paletteButton">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>70</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>调色板</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>370</y>
     <width>510</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>530</width>
     <height>345</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,150)</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>showPerArea</zorder>
  <zorder>bgTransparentSlider</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>mySkin</zorder>
  <zorder>netSkin</zorder>
  <zorder>bgTransparentSliderR</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>line</zorder>
  <zorder>customSkin</zorder>
  <zorder>paletteButton</zorder>
  <zorder>line_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicBackgroundListWidget</class>
   <extends>QListWidget</extends>
   <header>musicbackgroundlistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicBackgroundRemoteWidget</class>
   <extends>QWidget</extends>
   <header>musicbackgroundremotewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
